features/page_objects/journey/last_email_page.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require "json"
+
+class LastEmailPage < SitePrism::Page
+  # Page which shows last email sent to any address from the environment.
+  # As there are two app servers, there is a 50% chance that the latest email
+  # will show each time the page is refreshed.
+  # The page will be loaded up to 10 times until the email shows
+  # (a 1 in 1024 chance of the email not showing).
+
+  element(:email_content, "pre")
+
+  # Copy additional functions from WEX tests as needed:
+  # https://github.com/DEFRA/waste-exemptions-acceptance-tests/blob/master/features/page_objects/email/last_email_api_page.rb
+
+  def get_page_text(expected_string)
+    # Checks that the page contains a string unique to what's expected
+    # and returns the whole text of the page.
+
+    10.times do
+      page.evaluate_script "window.location.reload()"
+      page_text = email_content.text
+      next unless page_text.include?(expected_string)
+
+      return page_text
+    end
```

Comment:
Suggestion:
```
    10.times do
      page_text = email_content.text

      return page_text if page_text.include?(expected_string)

      page.evaluate_script "window.location.reload()"
    end
```
This will make sure that you only reload the page after you checked the content. The current version will reload the first page without checking the text. 
Also, in general, working with `if` rather than `unless` makes the code more readable :) 
Is a small change but I think it will give a little more chances to get the right email :)